docs/community/content/contribute/committer.cn.md

```diff
@@ -22,17 +22,47 @@ ShardingSphere社区遵循[Apache Community’s process](http://community.apache
 
 ## 日常工作
 
-1. Committer需要每天查看社区待处理的Pull Request和Issue列表，负责问题的处理：
+1. 每周负责轮值的 Committer 需要每天查看社区待处理的Pull Request 和 Issue 列表，负责问题的处理：
```

Comment:
Here is another.

Reply:
Hey @yx9o , I have redone it using the new merged branch, please review it once now.